tripper/mappings/mappings.py

```diff
@@ -277,11 +277,12 @@ def eval(
         inputs, idx = self.get_inputs(routeno)
         values = get_values(inputs, idx, quantity=quantity)
 
-        if len(inputs) == 1 and all(
-            isinstance(v, Value) for v in inputs.values()
-        ):
-            (value,) = tuple(inputs.values())
-        elif self.function:
+        # if len(inputs) == 1 and all(
+        #    isinstance(v, Value) for v in inputs.values()
+        # ):
+        #    (value,) = tuple(inputs.values())
+        # elif self.function:
```

Comment:
Why to keep the commented out code?

Reply:
I would like to make sure that things actually work as expected before removing the commented out lines, since they might address a special case I didn't thought about when creating this PR. The best check for that is DLite PR [#639](https://github.com/SINTEF/dlite/pull/639), which requires that this PR is merged first.